docroot/profiles/custom/cgov_site/modules/custom/cgov_home_landing/config/install/field.field.paragraph.ncids_imageless_card_group.field_card_layout_display.yml

```diff
@@ -0,0 +1,20 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.paragraph.field_card_layout_display
+    - paragraphs.paragraphs_type.ncids_imageless_card_group
+  module:
+    - options
+id: paragraph.ncids_imageless_card_group.field_card_layout_display
+field_name: field_card_layout_display
+entity_type: paragraph
+bundle: ncids_imageless_card_group
+label: field_card_layout_display
```

Comment:
@dlescarbeau This needs to be updated to be:
```
label: 'Card Layout Display'
```

Once you change this you can do a `drush features:import cgov_home_landing` to import the config so you can see the label in the front-end. (Know what the label is because of the content model on https://github.com/NCIOCPL/cgov-digital-platform/wiki/Paragraph-Type-NCIDS-Imageless-Card-Group#card-layout-display)